EnglishFilter/sections/antiadblock.txt

```diff
@@ -572,6 +572,9 @@ uaclips.com,kzclip.com,ru-clip.com,ruclip.com###morev
 !
 !
 !
+! https://github.com/AdguardTeam/AdguardFilters/issues/78720
+filmiseriali.com#%#//scriptlet("set-constant", "noAdBlock", "true")
+@@||filmiseriali.com/templates/filmiseriali/adblock/fuckadblock.min.js$domain=filmiseriali.com
```

Comment:
It's a Bulgarian site.
Move them to `foreign.txt` -> `Bulgarian` section
```
! https://github.com/AdguardTeam/AdguardFilters/issues/78720
filmiseriali.com#%#//scriptlet("set-constant", "noAdBlock", "true")
@@||filmiseriali.com/templates/filmiseriali/adblock/fuckadblock.min.js$domain=filmiseriali.com
```

Reply:
For some reasons, I have a problem adding rules to correct places.